lib/BlockedBy.js

```diff
@@ -69,6 +117,11 @@ class BlockedBy extends NodeResque.Plugin {
         const enforceBlockedBySelf = String(this.options.blockedBySelf) === 'true'; // because kubernetes value is a string
         const shouldDelete = await this.queueObject.connection.redis.get(deleteKey);
         const runningBuildId = await this.queueObject.connection.redis.get(runningKey);
+        const enableCollapse = String(this.options.collapse) === 'true'; // because kubernetes value is a string
+        const buildconfig = await this.queueObject.connection.redis.hget(
```

Comment:
camel case might be better
```suggestion
        const buildConfig = await this.queueObject.connection.redis.hget(
```

Reply:
fixed!